### PR TITLE
[FIX][account_anglo_saxon] Discount not included when comparing valua…

### PR DESCRIPTION
**Pull request by andrius-preimantas**
…tion price with invoice price

Steps to reproduce:
1. Create product: Stockable, Average Price (preferred); Real Time (automated) inventory valuation; Input, Output, Stock and Price Difference accounts all defined.
2. Create Purchase Order with price 10€
3. Validate Purchase Order and transfer Shipment
4. Based on Invoicing Control create Invoice either from PO/PO Line/Shipment
5. Add 10% discount on Invoice and validate Invoice

Current behavior:
- Account entry on receiving product created for 10€
- Account entry on Invoice Validation created for 9€

Expected behavior:
- Account entry on receiving product created for 10€
- Account entry on Invoice Validation created for 10€ including price difference line

In short, if PO and Supplier Invoice has the same price unit, but discount is defined, price difference is never evaluated.

Cheers
Andrius

### CODE REVIEW

**Comment by C3POdoo** [comment]
Hello,

This is an automatic message.

Thank you for your contribution but the version 8.0 is no longer supported. We only support the last 3 stable versions so no longer accepts patches into this branch.
We apology if we could not look at your request in time.

If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

Regards,
